src/pcapi/alembic/versions/20210629_3c46248f7143_educational_institution_add_adageid_.py

```diff
@@ -0,0 +1,25 @@
+"""educational_institution_add_adageId_field
+
+Revision ID: 3c46248f7143
+Revises: bb6bf7b9cdbd
+Create Date: 2021-06-29 15:38:50.855795
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "3c46248f7143"
+down_revision = "bb6bf7b9cdbd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column("educational_year", sa.Column("adageId", sa.String(length=30), nullable=True))
```

Comment:
```suggestion
    op.add_column("educational_year", sa.Column("adageId", sa.String(length=30), nullable=False))
```

Je ne pense pas qu'on ait envie que cette colonne soit vide

Reply:
Yes bien vu :)